bitnami/etcd/values.yaml

```
@@ -725,11 +727,17 @@ metrics:
   ## @param metrics.enabled Expose etcd metrics
   ##
   enabled: false
+
+
+  ## @param metrics.useSeparateEndpoint Use a separate endpoint for exposing metrics
+  #
+  useSeparateEndpoint: false
+
```

Going by the style of the rest of the file, there is no need for newlines between parameters of a subgroup:

```suggestion
  ## @param metrics.useSeparateEndpoint Use a separate endpoint for exposing metrics
  ##
  useSeparateEndpoint: false
```